js/components/adPanel.js

```diff
@@ -72,7 +72,16 @@ var AdPanel = React.createClass({
       adPlaybackInfo = adPlaybackInfo + ": (" + currentAdIndex + "/" + totalNumberOfAds + ")";
     }
 
-    var remainingTime = Utils.formatSeconds(Math.max(0, parseInt(this.props.adVideoDuration - this.props.currentPlayhead)));
+    var isLive = this.props.currentAdsInfo.currentAdItem.isLive;
```

Comment:
need a big old if check. I know it's not being done earlier in the function but we should start adding it in.